app/src/main/java/io/homeassistant/companion/android/bluetooth/ble/IBeaconTransmitter.kt

```diff
@@ -0,0 +1,11 @@
+package io.homeassistant.companion.android.bluetooth.ble
+
+data class IBeaconTransmitter(
```

Comment:
Ah - this is specifically an iBeaconTransmitter class rather than an interface.  Can change anyway though as agree is confusing (some premature optimisation going anticipating a move to multiple beacon types).

Reply:
Okay, I've decided to leave this as IBeacon for now, to make it explicit that's what it is (and also as the library I'm using already has a BeaconTrasnmitter class)- can change if easily enough if the implication of an interface is too strong.